Tools/autotest/arducopter.py

```diff
@@ -158,38 +158,38 @@ def land_and_disarm(self, timeout=60):
         self.wait_disarmed()
 
     def hover(self, hover_throttle=1500):
-        self.set_rc(3, hover_throttle)
+        self.set_throttle(hover_throttle)
     
     #Climb/descend to a given altitude
     def setAlt(self, desiredAlt=50):
         pos = self.mav.location(relative_alt=True)
         if pos.alt > desiredAlt:
-            self.set_rc(3, 1300)
+            self.set_throttle(1300)
             self.wait_altitude((desiredAlt-5), desiredAlt, relative=True)
         if pos.alt < (desiredAlt-5):
-            self.set_rc(3, 1800)
+            self.set_throttle(1800)
             self.wait_altitude((desiredAlt-5), desiredAlt, relative=True)
         self.hover()
     
     # Takeoff, climb to given altitude, and fly east for 10 seconds
     def takeoffAndMoveAway(self, dAlt=50, dDist=50):
         self.progress("Centering sticks")
-        self.set_rc(1, 1500)
-        self.set_rc(2, 1500)
-        self.set_rc(3, 1000)
-        self.set_rc(4, 1500)
+        self.set_roll(1500)
```

Comment:
I'm uneasy with this naming.

`set_roll` could be tagging any number of things as parameters.

I'm also actually rather uneasy with abstracting this too much.  For example, done properly `set_rc_pitch(pwm)` would go and check the rcmap and use that to set the appropriate channel.  That's obviously not going to happen (to expensive).

I really think that unless you can show a distinct advantage with the abstraction we shouldn't move from `set_rc(num, value)`.  The recent flip example was very, very much an aberration and insufficient to warrant such a change.

Reply:
I agree that set_roll/pitch/yaw aren't great but you are the one that proposed them ^^.

My main issue is that isn't the first time I notice a rc number error leading to issue in test.
I guess I will close this for this time and bring it later when we got another error ! I will just get the removal of mavproxy raw cmd